### PR TITLE
Fix coverage report generation for git subprojects (lcov >= 1.10 required).

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

**Comment by eile** [comment]
Also enforce version in Coverage.cmake (mostly there already)

**Comment by rdumusc** [comment]
Yes, removed the gcc part of the version check to enforce lcov >= 1.10

File: Coverage.cmake [hunk 1]
```diff
@@ -60,18 +59,19 @@ macro(COVERAGE_REPORT)
     # Tweak coverage limits to yellow 40%/green 80%
     set(COVERAGE_LIMITS --rc genhtml_med_limit=40 --rc genhtml_hi_limit=80)
   endif()
+  message(STATUS, "Creating target for: ${PROJECT_NAME}, ${PROJECT_SOURCE_DIR}")
```
Comment (eile) [inline Coverage.cmake]: rm?

Comment (rdumusc) [inline Coverage.cmake]: Arrg, sorry! fixed.

**Pull request merged**